pkg/agent/cniserver/pod_configuration.go

```diff
@@ -442,9 +449,9 @@ func (pc *podConfigurator) reconcile(pods []corev1.Pod, containerAccess *contain
 			// interface should no longer be in store after the call to removeInterfaces
 		}
 	}
-
-	missingPods := desiredPods.Difference(actualPods)
-	pc.reconcileMissingPods(missingPods, containerAccess)
+	if len(missingIfConfigs) > 0 {
+		pc.reconcileMissingPods(missingIfConfigs, containerAccess)
```

Comment:
It seems `actualPods` is no longer needed after this change.

Reply:
Removed